core/__tests__/models/recordProperty/recordProperty.ts

```diff
@@ -269,19 +269,19 @@ describe("models/recordProperty", () => {
         });
         await recordProperty.setValue("4128889999");
         const response = await recordProperty.getValue();
-        expect(response).toBe("+1 412 888 9999");
+        expect(response).toBe("4128889999");
       });
 
-      test("invalid numbers are stored", async () => {
+      test("invalid phone numbers are stored without change", async () => {
         const recordProperty = new RecordProperty({
           recordId: record.id,
           propertyId: phoneNumberProperty.id,
         });
         await recordProperty.setValue("5");
         const response = await recordProperty.getValue();
-        expect(response).toBe(null);
-        expect(recordProperty.invalidValue).toBe("5");
-        expect(recordProperty.invalidReason).toBe("Invalid phoneNumber value");
+        expect(response).toBe("5");
+        expect(recordProperty.invalidValue).toBe(null);
+        expect(recordProperty.invalidReason).toBe(null);
```

Comment:
I think the spirit of the test has changed here - the phone number "5" is no longer invalid, because we've stopped validating it

Reply:
I can remove this test, but felt it was perhaps worth keeping in case we decide to add validation back at some later point.